### PR TITLE
TOOLS: ucc info print config variables

### PR DESCRIPTION
**Pull request by Sergei-Lebedev**
## What
Add -c option to ucc_info to print UCC config variables
CI runs ucc_info

## Why ?
Convenient way to print all config variables in UCC 

## How ?
ucs_config_print_all_opts is used. Changes in this PR requires latest version of UCS to work properly (https://github.com/openucx/ucx/pull/5876).
config tables of UCC separated from UCX config tables

### CODE REVIEW

**Comment by Sergei-Lebedev** [comment]
Also added -b option to print build configuration

**Comment by Sergei-Lebedev** [comment]
Also added -f option to print config values with description

**Review by alex--m — APPROVED** [review]
Very useful!

File: src/utils/ucc_datastruct.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCC_DATASTRUCT_H_
+#define UCC_DATASTRUCT_H_
+
```
Comment (vspetrov) [inline src/utils/ucc_datastruct.h:8]: @Sergei-Lebedev do we need "#include <ucs/datastruct/list.h>" here?

**Review by vspetrov — APPROVED** [review]

**Comment by manjugv** [comment]
Looks good!

**Review by manjugv — APPROVED** [review]

**Pull request merged**